lendingbot.py

```diff
@@ -181,16 +181,16 @@
 		pass
 sleepTime = sleepTimeActive #Start with active mode
 #sanity checks
-if sleepTime < 1 or sleepTime > 3600 or sleepTimeInactive < 1 or sleepTimeInactive > 3600:
-	print "sleeptime values must be 1-3600"
+if sleepTime <= 1 or sleepTime >= 3600 or sleepTimeInactive <= 1 or sleepTimeInactive >= 3600:
+	print "sleeptime values must be 1-3600 seconds"
 	exit(1)
-if minDailyRate < 0.00003 or minDailyRate > 0.05: # 0.003% daily is 1% yearly
+if minDailyRate <= 0.003 or minDailyRate >= 5: # 0.003% daily is 1% yearly
```

Comment:
IMHO the values here should be 0.00003 and 0.05 since when we read the values from the configuration file we divide by 100.  i.e. if you configure the minDailyRate in the config to 0.003 it will be 0.00003 at this point. 

See lines 103, 105, 164, 165